src/Tile/Tile.js

```diff
@@ -39,9 +40,9 @@ Tile.propTypes = {
     backgroundColor: PropTypes.number,
     className: PropTypes.string,
     colorAccent: PropTypes.number,
-    columnSpan: PropTypes.number,
+    columnSpan: PropTypes.oneOf(GRID_COLUMNS),
     disabled: PropTypes.bool,
-    rowSpan: PropTypes.number
+    rowSpan: PropTypes.oneOf(GRID_ROWS)
```

Comment:
I don't believe `rowSpan` should have a `oneOf` list of values.  Columns, yes, but not rows.  And I also agree with @bcullman about `TILE_GRID_COLUMNS`.